source/com.google.mlkit/image-labeling-custom/Transforms/Metadata.Namespaces.xml

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<metadata>
+
+  <attr
+      path="/api/package[@name='com.google.android.gms.internal.mlkit_vision_label_custom']"
+      name="managedName"
+      >
+      Android.Gms.Internal.MLKitVisionLabelCustom
```

Comment:
I realize Google is being inconsistent here, but would it be better for ***us*** to be consistent and use `Android.Gms.Internal.MLKit.Vision.Label.Custom`?  (Same with all the other ones.)